templates/16-04/azure.html

```diff
@@ -0,0 +1,145 @@
+      </h2>
+      <p>
+        If you would like to keep using Ubuntu 16.04 LTS, Canonical is delighted to offer at least three-additional years of Extended Security Maintenance (ESM) available with the <a class="p-link--external" href="https://azuremarketplace.microsoft.com/en-gb/marketplace/apps/canonical.0001-com-ubuntu-pro-xenial?tab=overview">Ubuntu Pro 16.04 LTS premium image</a> or through an <a href="/advantage">Ubuntu Advantage  subscription</a> using the <a class="p-link--external" href="https://discourse.ubuntu.com/t/ubuntu-advantage-client/21788">UA client</a>.
+      </p>
+      <p>
+        Ubuntu Pro 16.04 is priced at 3.0-4.5% of your average compute cost of your existing 16.04 LTS instances with hourly billing, or alternatively you can get an Ubuntu Advantage subscription starting at $75 per VM per year.
+      </p>
+    </div>
+    <div class="col-5 u-align--center u-vertically-center u-hide--small">
+      {{
+        image(
+        url="https://assets.ubuntu.com/v1/da9a1344-Microsoft-Azure-logo_stacked_transparent.png",
+        alt="",
+        width="350",
+        height="100",
+        hi_def=True,
+        loading="auto",
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-8">
+      <h2>
+        Upgrading Ubuntu 16.04 LTS on Azure to Ubuntu Pro
+      </h2>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-6">
+      <h3 class="p-muted-heading">
+        New or redeployed instances
+      </h3>
+      <p>
+        For fresh Ubuntu 16.04 instances, or if you are able to redeploy your workload, we recommend using Ubuntu Pro images. <a href="/azure/pro">Ubuntu Pro for Azure</a> is a premium image, optimised for Azure with security and compliance features built in, and optimised for the public cloud.
+      </p>
+      <p>
+        Ubuntu Pro includes security coverage for the entire collection of software packages shipped with Ubuntu, tracking high and critical CVEs. It is ideal for companies that have embraced open source for their new products.
+      </p>
+      <p>
+        <a class="p-button--positive p-link--external" href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.0001-com-ubuntu-pro-xenial?tab=Overview">Launch Ubuntu Pro 16.04 LTS on Azure</a>
+      </p>
+    </div>
+    <div class="col-6">
+      <h3 class="p-muted-heading">
+        Upgrading existing instances
+      </h3>
+      <p>
+        If you are looking to upgrade your existing Ubuntu 16.04 LTS Azure instances you can <a class="p-link--external" href="https://azuremarketplace.microsoft.com/uk-ua/marketplace/apps/Canonical.UbuntuAdvantageSupport?tab=Overview">enable the Ubuntu Advantage subscription directly from the Azure marketplace</a>.
+      </p>
+      <p>
+        Please note that this will enable Ubuntu Advantage for all Ubuntu instances under the selected subscription.
+      </p>
+    </div>
```

Comment:
Missing paragraph here "If you are looking to upgrade your existing...."

Reply:
isn't that on line 102?

Reply:
ahh yes sorry missed that!